### PR TITLE
Prepare develop for release/v1.16.1

### PR DESCRIPTION
**Pull request by akoptelov**

### CODE REVIEW

File: apps/deploy_monitoring/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "deploy-monitoring"
-version = "1.17.0"
```
Comment (tizoc) [inline apps/deploy_monitoring/Cargo.toml:3]: @akoptelov `1.16.1` right?

Comment (akoptelov) [reply apps/deploy_monitoring/Cargo.toml:3]: Looks like it is not handled by `version.sh`. I'll fix that.

Comment (tizoc) [reply apps/deploy_monitoring/Cargo.toml:3]: Ok, same for the title of the PR (I had exactly the same issue when releasing 1.16.0)

Comment (akoptelov) [reply apps/deploy_monitoring/Cargo.toml:3]: Closing this one in favor of https://github.com/tezedge/tezedge/pull/970

**Pull request closed without merging**